gtfs/spec/en/reference.md

```diff
@@ -375,3 +375,23 @@ The file contains information about the dataset itself, rather than the services
 |  `feed_version` | Text | Optional | String that indicates the current version of their GTFS dataset. GTFS-consuming applications can display this value to help dataset publishers determine whether the latest dataset has been incorporated. |
 |  `feed_contact_email` | Email | Optional | Email address for communication regarding the GTFS dataset and data publishing practices. `feed_contact_email` is a technical contact for GTFS-consuming applications. Provide customer service contact information through [agency.txt](#agencytxt). |
 |  `feed_contact_url` | URL | Optional | URL for contact information, a web-form, support desk, or other tools for communication regarding the GTFS dataset and data publishing practices. `feed_contact_url` is a technical contact for GTFS-consuming applications. Provide customer service contact information through [agency.txt](#agencytxt). |
+
+### attributions.txt
+
+File: **Optional**
+
+The file defines the attributions applied to the dataset.
+
+|  Field Name | Type | Required | Description |
+|  ------ | ------ | ------ | ------ |
+|  `attribution_id` | ID | Optional | Identifies an attribution for the dataset or a subset of it. This is mostly useful for translations. |
+|  `agency_id` | ID referencing `agency.agency_id` | Optional | Agency to which the attribution applies.<br><br>If one `agency_id`, `route_id`, or `trip_id` attribution is defined, the other ones must be empty. If none of them is specified, the attribution will apply to the whole dataset. |
+|  `route_id` |  ID referencing `routes.route_id`  | Optional | Functions in the same way as `agency_id` except the attribution applies to a route. Multiple attributions can apply to the same route. |
+|  `trip_id` |  ID referencing `trips.trip_id`  | Optional | Functions in the same way as `agency_id` except the attribution applies to a trip. Multiple attributions can apply to the same trip. |
+|  `organization_name` | Text | **Required** | Name of the organization that the dataset is attributed to. |
+|  `is_producer` | Enum | Optional | The role of the organization is producer. Valid options are:<br><br>`0` or empty - Organization doesn’t have this role.<br>`1` - Organization does have this role.<br><br>At least one of the fields `is_producer`, `is_operator`, or `is_authority` should be set at `1`. If none of them is set to `1` (e.g. because they are non-official fields defining other types of attribution), the data consumer is allowed to ignore them. |
```

Comment:
I don't understand the `data consumer is allowed to ignore them`. What is the goal of this sentence ?
Does allowing / forbidding something to the data consumer should be stated in the spec ? I think it depends on the chosen license attached to the feed, and it's not included in the GTFS (yet ?).

Reply:
My phrasing was bad, and my intend wasn't the one you're describing. But I think you're right on the point that the spec can just stay silent on that case.